userguide/payment/includes/payment-flows.adoc

```diff
@@ -66,21 +66,28 @@ The conceptual flow below shows a 3D-Secure variation of the previous flows. The
+There are two main types of scenarios:
 
-Before any payment operation can occur, the user must first enter his payment information. In Kill Bill, this translates into creating a PaymentMethod. When dealing with credit card information specifically, merchants have to keep the data into a PCI compliant environment (often handled by the gateway): this process requires tokenization of the PAN, so that the payment system only handles the token, which is stored at the plugin level and passed to Kill Bill as a plugin property. Take a look at our https://github.com/killbill/killbill-stripe-demo[Stripe demo] for a concrete example.
-
-Note that the creation of the Kill Bill payment method is not represented in the flows below: it can happen either in a prior step or during the payment using one of our combo payment call. Usually, the user is prompted with a form to enter his payment method information, and then the payment is initiated using one of the *createAuthorization* or *createPurchase* APIs. Kill Bill will connect to its payment plugin which in turn will contact the gateway to perform the required operation. Upon success/failure, the user is redirected to a landing page.
-
-.Gateway integration (direct payment API)
+* When payment gateways provide synchronous APIs, transactions end up in a terminal state right away. The following figure demonstrates this scenario:
++
+.Gateway integration (With Synchronous Payment API)
 
 image::https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/img/payment-userguide/payments_gateway.png[width=960,align="center"]
++
+The payment is initiated using one of the *createAuthorization* or *createPurchase* APIs. Kill Bill then connects to its payment plugin which in turn contacts the gateway to perform the required operation. Upon success/failure, the user is redirected to a landing page.
++
+[NOTE]
+*Note:*
+Before any payment operation can occur, the user must first enter his/her payment information and a payment method must be created as explained in the <<payment_method_flows, payment method flows>> section. This is not represented in the flow above: it can happen either in a prior step or during the payment using one of our https://killbill.github.io/slate/#payment-combo-api-to-create-a-new-payment-transaction-on-a-existing-or-not-account[combo payment call]. 
```

Comment:
```suggestion
Before any payment operation can occur, the user must first enter his/her payment information and a payment method must be created as explained in the <<payment_method_flows, payment method flows>> section. This is not represented in the flow above; it can happen either in a prior step or during the payment using one of our https://killbill.github.io/slate/#payment-combo-api-to-create-a-new-payment-transaction-on-a-existing-or-not-account[combo payment call]. 
```